csvqb/csvqb/configloaders/infojson1point1/proposed-schema.json

```diff
@@ -0,0 +1,893 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "http://gss-cogs.github.io/family-schemas/dataset-schema-1-1-0.json",
```

Comment:
Are we happy with the URI I'm proposing for the new schema document, @canwaf @rossbowen ?

Reply:
Yup. This looks fine to me.

Reply:
I'd normally go for something like `dataset-schema-1.1.0.json`, but this is fine. Ultimately it'll be good to put them somewhere stable (e.g. reference.data.gov.uk/schemas or some such) and also agree on a policy for versioning.